### ISSUE TITLE
Effective Learning Rate Governs Loss Dynamics in Language Model Pretraining

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#546]
## Links
https://arxiv.org/abs/2608.24814

<img width="900" height="498" alt="Image" src="https://github.com/user-attachments/assets/7ea0ef99-e007-4d83-b66a-b865ed4712fd" />

## Summary

LLMの学習損失は、learning rateそのものではなく、

$$ \mathrm{ELR}_t=\frac{\eta_t}{\|W_t\|_F} $$

というEffective Learning Rateによって主に決まる。

なぜノルムで割るのか

同じLRでも、パラメータノルムが大きくなると相対的な更新は小さくなります。

$$ \frac{\|\Delta W\|}{\|W\|} \approx \frac{\eta}{\|W\|} $$

そのためAdam、Muon、Signumのように更新量が正規化されるoptimizerでは、raw LRよりもELRの方が実際の学習速度をよく表します。

主要な実験結果

著者らは、LRとパラメータノルムを大きく変えながら、ELRだけを同じにしました。するとloss curveがほぼ一致しました。

Llama、Qwen3、MoE、Kimi Delta Attentionで成立
AdamW、Muon、Signumで成立
最大1Bモデルまで確認
平均loss差はおおむね \(2\)–\(5\times10^{-3}\)

これを「ELR collapse」と呼んでいます。

Weight decayとHyperballの解釈

Weight decayは単なる正則化ではなく、

$$ \|W_t\|\text{の増加を抑える} \Rightarrow \mathrm{ELR}_t\text{を高く維持する} $$

という役割を持つと解釈できます。

Hyperballも特殊な仕組みというより、特定のELR scheduleを実現するnorm-control手法と見なせます。

Delayed acceleration

Weight decayありのrunが、序盤では悪くても終盤で追い越す現象もELRで説明しています。

高いELRを維持するとsignalの学習は速い
同時にnoiseも増えるため、序盤のlossは悪く見える
終盤にELRをdecayするとnoiseが減り、先に獲得した利点が現れる

つまり「利得は早期に獲得され、終盤に観測される」という説明です。

重要な限界
主に124M、最大1Bなので大規模LLMでは未検証
training lossのみで、validationや下流性能は未確認
QK-Normとlearnable RMSNorm gainがないと精度が悪化
LRとnormを高速に変えるとELR collapseも崩れる
batch sizeは固定されており、batch ramp-upについては何も結論できない

要するに、この論文の価値は、

LR、weight decay、norm constraintを別々に設計するのではなく、まず望ましいELR scheduleを決める

という新しい学習設計の見方を提示した点です。
